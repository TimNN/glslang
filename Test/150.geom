#version 150 core

void main()
{
    EmitVertex();
    EndPrimitive();
    EmitStreamVertex(1);    // ERROR
    EndStreamPrimitive(0);  // ERROR
}
