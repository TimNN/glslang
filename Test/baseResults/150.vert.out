Warning, version 150 is not yet complete; some version-specific features are present, but many are missing.
ERROR: 0:13: 'gl_ClipDistance' : undeclared identifier 
ERROR: 0:13: 'gl_ClipDistance' :  left of '[' is not of type array, matrix, or vector  
ERROR: 0:13: 'assign' :  l-value required (can't modify a const)
ERROR: 0:16: 'gl_' : reserved built-in name 
ERROR: 0:21: 'a' : cannot redeclare a user-block member array 
ERROR: 5 compilation errors.  No code generated.

ERROR: node is still EOpNull!
0:9  Function Definition: main( (void)
0:9    Function Parameters: 
0:11    Sequence
0:11      move second child to first child (4-component vector of float)
0:11        'gl_Position' (invariant gl_Position 4-component vector of float)
0:11        'iv4' (in 4-component vector of float)
0:12      move second child to first child (float)
0:12        'gl_PointSize' (gl_PointSize float)
0:12        'ps' (uniform float)
0:13      move second child to first child (float)
0:13        Constant:
0:13          0.000000
0:13        direct index (float)
0:13          'iv4' (in 4-component vector of float)
0:13          Constant:
0:13            0 (const int)
0:?   Linker Objects
0:?     'iv4' (in 4-component vector of float)
0:?     'ps' (uniform float)
0:?     'gl_ClipDistance' (4-element array of float)
0:?     '__anon__0' (layout(shared ) uniform block)
0:?     'gl_VertexID' (gl_VertexId int)
0:?     'gl_InstanceID' (gl_InstanceId int)

